pyproject.toml

```diff
@@ -0,0 +1,70 @@
+[tool.poetry]
+name = "scoap3"
+version = "0.1.0"
+description = ""
+authors = ["CERN SIS-TS <cern-sis-ts-admins@cern.ch>"]
+license = "MIT License"
+
+[tool.poetry.dependencies]
+python = "~3.10"
+pytz = "^2023.3"
+python-slugify = "^8.0.1"
+Pillow = "^9.5.0"
+argon2-cffi = "^21.3.0"
+whitenoise = "^6.4.0"
+redis = "^4.5.4"
+hiredis = "^2.2.2"
+celery = "^5.2.7"
+django-celery-beat = "^2.5.0"
+flower = "^1.2.0"
+Django = "^4.2"
+django-environ = "^0.10.0"
+django-model-utils = "^4.3.1"
+django-allauth = "^0.54.0"
+django-crispy-forms = "^2.0"
+crispy-bootstrap5 = "^0.7"
+django-redis = "^5.2.0"
+djangorestframework = "^3.14.0"
+django-cors-headers = "^3.14.0"
+drf-spectacular = "^0.26.1"
+django-webpack-loader = "^1.8.1"
+gunicorn = {version = "^20.1.0", extras = ["production"]}
+psycopg2 = {version = "^2.9.6", extras = ["production"]}
+sentry-sdk = {version = "^1.19.1", extras = ["production"]}
+django-storages = {extras = ["production"], version = "^1.13.2"}
+django-anymail = {version = "^9.1", extras = ["production"]}
```

Comment:
1. We'll have just one version of requirements (not local & production)
2. where are `[tool.poetry.extras]`?

Reply:
> * where are `[tool.poetry.extras]`?

What do we need them for?

Reply:
it's ok now, you put `{extras = ["production"], version = "^1.13.2"}`, so then you need to have `[tool.poetry.extras]` section